jobs/workflow_test.groovy

```diff
@@ -64,8 +63,20 @@ import utilities.StatusUpdater
                  condition {
                    status(buildStatus, buildStatus)
                    steps {
-                     shell StatusUpdater.updateStatus(
-                       commitStatus: commitStatus, repoName: '${COMPONENT_REPO}', commitSHA: '${ACTUAL_COMMIT}', description: "${name} job ${buildStatus}")
+                     shell """
+                       #!/usr/bin/env bash
+                       set -eo pipefail
+
+                       curl \
+                       --silent \
+                       --user deis-admin:"\${GITHUB_ACCESS_TOKEN}" \
+                       --data '{ \
+                       "state":"${commitStatus}", \
+                       "target_url":"'"\${BUILD_URL}"'", \
+                       "description":"${name} job ${buildStatus}", \
+                       "context":"ci/jenkins/pr"}' \
+                       "https://api.github.com/repos/deis/\${COMPONENT_REPO}/statuses/\${ACTUAL_COMMIT}"
+                     """.stripIndent().trim()
```

Comment:
Looks like maybe this is part of some conflicts/new stuff that's dropped in master since PR was created? Rebase should help?